### PR TITLE
ObjectData View 

### PR DESCRIPTION
**Pull request by HassanAlhawari1998**
Currently, you can only create, edit, and delete property data. Other data such as buildings, apartments, and garages are represented by dummy data. #137
ObjectDataView contains all objectData
image
PropertyView contains PropertyData, which can be updated or deleted depending on the route
image

### CODE REVIEW

**Comment by HassanAlhawari1998** [comment]
Created an Object Data View to address issue #137

**Comment by HassanAlhawari1998** [comment]
@astanik  I have built the basic structure to update, create, and delete object data. The API requests for apartment and garages can be implemented once the endpoints are ready.

There will be improvements in performance and code, but first, I would like someone to review this pull request.

File: src/views/PropertyView.vue [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+            @click="createProperty"
+            class="p-button-property mr-2 mb-2"
+          />
+          <Button
+            v-if="action === 'update'"
+            label="Updaten"
+            icon="pi pi-check"
+            @click="updateProperty"
+            class="p-button-property mr-2 mb-2"
+          />
```
Comment (Tobiwue) [inline src/views/PropertyView.vue:203]: for qol you could notify the user that updating the data worked correctly
this also applies for all the other buttons

File: src/views/ObjectDataView.vue [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+            err.message || "Unknown error"
+          }`;
+          return [];
+        });
+    }
+
+    function generateDummyData(properties: PropertyItem[]) {
+      const dummyBuildings = generateDummyBuildings(properties);
+      const dummyApartments = generateDummyApartments(dummyBuildings);
+      const dummyGarages = generateDummyGarages(dummyBuildings);
```
[the remark this answers is not in the document]
Comment (HassanAlhawari1998) [reply src/views/ObjectDataView.vue:42]: Yes, the endpoints will be added later

File: src/views/PropertyView.vue (hunk 1 above)
Comment (HassanAlhawari1998) [reply src/views/PropertyView.vue:203]: Great Idea. This should be implemented

**Pull request merged**